ingest-app/src/cmr/ingest/data/bulk_update.clj

```diff
@@ -13,12 +13,31 @@
    [cmr.oracle.sql-utils :as su]))
 
 (defn generate-task-status-message
-  "Generate overall status message based on number of collection failures."
-  [num-failed-collections num-total-collections]
-  (if (= 0 num-failed-collections)
+  "Generate overall status message based on number of collection failures and skips."
+  [num-failed-collections num-skipped-collections num-total-collections]
+  (if (and (= 0 num-failed-collections)
+           (= 0 num-skipped-collections))
     "All collection updates completed successfully."
-    (format "Task completed with %d collection update failures out of %d"
-      num-failed-collections num-total-collections)))
+    (let [failed-msg (when (not= 0 num-failed-collections)
+                       (str num-failed-collections " FAILED"))
+          skipped-msg (when (not= 0 num-skipped-collections)
+                        (if (some? failed-msg)
+                          (str ", " num-skipped-collections " SKIPPED")
+                          (str num-skipped-collections " SKIPPED")))
+          updated-msg (let [num-updated-collections (- num-total-collections
+                                                       (+ num-failed-collections
+                                                          num-skipped-collections))]
+                        (when (not= 0 num-updated-collections)
+                          (if (or (some? failed-msg) (some? skipped-msg))
+                            (str " and " num-updated-collections " UPDATED")
+                            (str num-updated-collections " UPDATED"))))]
+      (str "Task completed with " 
+           failed-msg 
+           skipped-msg
+           updated-msg 
+           " out of totally " 
```

Comment:
This phrasing is a little unusual (it uses an adverb where an adjective is expected); it would probably be better to say " ... of <num> total collection update(s)".

Reply:
Thanks. Done.